sdk/core/AzureCore/Source/Pipeline/Policies/AuthenticationPolicy.swift

```diff
@@ -24,16 +24,21 @@
 //
 // --------------------------------------------------------------------------
 import Foundation
-import MSAL
 
 public typealias TokenCompletionHandler = (AccessToken?, Error?) -> Void
-public typealias MSALResultCompletionHandler = (MSALResult?, Error?) -> Void
 
 public struct AccessToken {
     // MARK: Properties
 
     public let token: String
     public let expiresOn: Int
+
+    // MARK: Initializers
+
+    public init(token: String, expiresOn: Int) {
```

Comment:
Since this is a struct, an initializer of this sort should be a freebie and you shouldn't need to code it.

Reply:
I thought so too, but the auto-created initializer wasn't public - I was unable to access it from other projects. Was very confused but this got it going. I will investigate further.